.github/workflows/docs-ci.yml

```diff
@@ -0,0 +1,41 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+name: "Docs CI"
+on:
+  push:
+    branches:
+      - 'main'
+      - '1.*'
+      - '2.*'
+    tags:
+      - 'apache-iceberg-**'
+  pull_request:
+    paths:
+      - '**.md'
+      - '.github/workflows/docs-ci.yml'
+
+jobs:
+  lint-markdown-files:
```

Comment:
Can we have 1 job that runs markdown for all md files? Instead of including this in multiple workflows

Reply:
Note this job in other workflows are added to `paths-ignore`.